getHacks.js

```diff
@@ -18,16 +18,13 @@ export const list = async (event, context) => {
     });
     conn.model(
       "Hack",
-      new mongoose.Schema({ title: String, description: String, goal: String })
+      new mongoose.Schema({ title: String, description: String, goal: String, team: Array })
     );
+    conn.model("User", new mongoose.Schema({ name: String }));
   }
   const Query = conn.model("Hack");
-  /**
-   * To be used in the future This will return One document
-   * const doc = await Query.findOne({_id: '5e6094446a56971ad6a32d7b'});
-   */
   try {
-    const doc = await Query.find();
+    const doc = await Query.find().populate('team', '-email', 'User');
```

Comment:
I thought mongoose would only populate what was in the schema, and you wouldnt need to redact the email field.

Reply:
I had the same thought, however it still grabs the email